cookbooks/bcpc-hadoop/Gemfile

```diff
@@ -1,6 +1,16 @@
 ruby RUBY_VERSION
+if File.exist?('/opt/chef/Gemfile')
```

Comment:
this file no longer exists. let's stop doing this.

Reply:
Removed